hs_app_timeseries/tests/test_timeseries_metadata.py

```diff
@@ -39,14 +40,15 @@ def setUp(self):
         )
 
         self.temp_dir = tempfile.mkdtemp()
-        self.odm2_sqlite_file_name = 'ODM2_valid.sqlite'
```

Comment:
@pkdash Now that this test file ```ODM2_valid.sqlite``` is not used any more, the test file should be deleted from the test directory.